protocols/ledger/ledger.go

```diff
@@ -15,8 +15,8 @@ import (
 type LedgerManager struct {
 }
 
-func NewLedgerManager() LedgerManager {
-	return LedgerManager{}
```

Comment:
This was done so that we could easily get a pointer to the `LedgerManager` in the engine. Although due to the Ledger Managers simplicity at the moment, I don't think it matters too much if we have a pointer or a direct reference to it in the engine.